python/sglang/srt/configs/exaone.py

```diff
@@ -0,0 +1,193 @@
+# coding=utf-8
+# Copyright 2021 The LG AI Research EXAONE Lab. All rights reserved.
```

Comment:
nit: 2021 -> 2024
And may you add `Copyright 2024 SGLang Team` here?

Reply:
Okay, I'll add Copyright 2023-2024 SGLang Team

Reply:
@zhyncs 
nit: 2021 -> 2024? what is this mean?

Reply:
Completed adding "Copyright 2023-2024 SGLang Team" to "python/sglang/srt/configs/exaone.py"!

Reply:
Why is the copyright on your team's work marked as 2021?

Reply:
aha!
I added two more like this.
# Copyright 2024 The LG AI Research EXAONE Lab. All rights reserved.
- Revised 2021 to 2024,
# Copyright 2024 The LG CNS AI Engineering Team.
- Added LG CNS COPYRIGHT.

LG AI Research and LG CNS are related to the same LG affiliate, both companies are related to the EXAONE model. For your information, I belong to LG CNS. 
Thank you!